front/app/containers/SearchPage/SearchPage.tsx

```diff
@@ -502,43 +502,36 @@ class SearchPage extends React.Component<SearchPageProps, SearchPageState> {
   };
 
   componentDidMount() {
-    let searchTerm = this.props.location.search
-    searchTerm = searchTerm.slice(2)
-    searchTerm = searchTerm.replace(/\+/g, " ")
-    if (this.state.showCards) {
-      window.addEventListener('scroll', this.handleScroll);
-      if(this.props.location.search){
-        let q = {key:"AND",children:[{children: [], key: searchTerm}]} 
-        this.setState({
+    let searchTerm = this.props.location.search;
+    searchTerm = searchTerm.slice(2);
+    searchTerm = searchTerm.replace(/\+/g, ' ');
```

Comment:
I've never seen the query string as `?=terms`  It's much more common to format is as `field1=value1&field2=value2&field3=value3` which also allows the potential for more terms to be added in the future.  The slice(2) also makes a lot of assumptions about the url for something that comes from off of the site. I think we should be more defensive and split out the terms.  

I suggest q= for the search query.  Some day we might add the ability to add agg filters too and this will preserve backwards compat.

Reply:
When you navigate to https://clinwiki.wpengine.com/ and search for COPD for example, the wpengine site redirects you to https://www.clinwiki.org/?=COPD
So I was working under the assumption that was intended behavior specifically for the wordpress site

Reply:
@williamhoos is this something we can easily configure on the wordpress side?

Reply:
I' m sure we can change anything. Just tell me what wordpress should send. Perfect example of don't assume something has to be a certain way. What are the "requirements" for ideally how WP submits values (I assume it will escape spaces with ASCII type codes too. I can get it changed or Dan/ @hifilorau  are connected with the website guys (Chris at Tomatillo)

Reply:
I think ideally insead of ?=query we'd have ?q=query.  I would guess wordpress will escape the string for use in a url automatically.